--- a/examples/simple_room_server/main.cpp
+++ b/examples/simple_room_server/main.cpp
@@ -3,6 +3,10 @@
 
 #include "MyMesh.h"
 
+#ifdef WITH_POTA_GATEWAY
+#include "pota_spotter.h"
+#endif
+
 #ifdef ETHERNET_ENABLED
   #define ETHERNET_CLI_BANNER "MeshCore Room Server CLI"
   #include <helpers/nrf52/EthernetCLI.h>
@@ -87,6 +91,10 @@
 
   the_mesh.begin(fs);
 
+#ifdef WITH_POTA_GATEWAY
+  PotaSpotter::initWiFi();
+#endif
+
 #ifdef DISPLAY_CLASS
   ui_task.begin(the_mesh.getNodePrefs(), FIRMWARE_BUILD_DATE, FIRMWARE_VERSION);
 #endif
@@ -157,4 +165,7 @@
 #ifdef HAS_EXTERNAL_WATCHDOG
   external_watchdog.loop();
 #endif
+#ifdef WITH_POTA_GATEWAY
+  PotaSpotter::handleLoop();
+#endif
 }
